mitodl/ocw-studio
# import video galleries

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-studio/issues/847

#### What's this PR do?
In https://github.com/mitodl/ocw-to-hugo/pull/395, we set up `ocw-to-hugo` to import pages with `is_media_gallery` set to True as a video gallery and place it under `content/video_galleries.  This PR alters the `ocw_import` code to take this into consideration and mark video gallery pages to be of the `video_gallery` type.

#### How should this be manually tested?
 - Make sure your local instance of `ocw-studio` has S3 credentials for downloading legacy OCW courses for import
 - Spin up a local instance and in a separate terminal run `docker-compose run web ./manage.py import_ocw_course_sites --bucket ocw-to-hugo-output-qa --filter es-s41-speak-italian-with-your-mouth-full-spring-2012`
 - Visit http://localhost:8043/sites/es-s41-speak-italian-with-your-mouth-full-spring-2012/type/video_gallery/ and ensure that you see "Video Lectures" imported there and can click on it and see all of the included videos
